.github/workflows/build.yml

```diff
@@ -53,6 +53,30 @@ jobs:
         with:
           submodules: recursive
 
+      - name: Extract branch name
+        id: branch_env
+        shell: bash
+        run: |
+          export VERSION=${GITHUB_REF##*/}
+          echo "##[set-output name=version;]$(echo ${VERSION:1})"
+          echo "##[set-output name=fullname;]$(echo apache-apisix-${VERSION:1}-src.tgz)"
+
+      - name: Create tarball
+        # if: ${{ startsWith(github.ref, 'refs/heads/v') && !endsWith(matrix.os_name, 'luarocks') }}
+        run: |
+          export VERSION=${{ steps.branch_env.outputs.version }}
+          export RELEASE_SRC=${{ steps.branch_env.outputs.fullname }}
+          make compress-tar
+          # mv release/${RELEASE_SRC} ${RELEASE_SRC}
+          ls
+
+      - name: Remove everything except tarball/.travis/t
+        # if: ${{ startsWith(github.ref, 'refs/heads/v') && !endsWith(matrix.os_name, 'luarocks') }}
```

Comment:
Are we changing the release name these days? Yeah it would definitely be of help, so people won't trigger this action when push to their own branch say `verbose`.

And to the last problem, since it would only be triggered when push to release branches, maintainers could control who would trigger this CI.

Reply:
We don't need to change the release name. We only need to change the branch name.

Will you submit a final commit to `release/test` branch and show us the final result after changing this? Thanks!